docs/basic_construction.md

```diff
@@ -0,0 +1,125 @@
+# Telescope - Basic Construction
+**ALBA** provides a method for a prover to convince a verifier that the prover knows at more than $n_f$ elements satisfying a predicate $R$, while only presenting a small subset of elements as proof. 
+This construction uses a recursive filtering process, guided by random oracles, to produce succinct and efficient proofs.
+
+## Overview
+- The prover builds a proof by creating tuples (sequences) of elements from $S_p$, the set of elements. 
+- The process begins with roots as starting points and expands them step-by-step. 
```

Comment:
Perhaps rewrite this with "The tuple are initilised with ..." ?
The term "roots" here is a bit confusing as you haven't talked about trees yet.